### PR TITLE
Mention the status of `await` inside of modules

### PR DESCRIPTION
**Pull request by jkrems**
Tried to also cram in a mention of "more differences when compared to CommonJS" but couldn't find a good way to fit that into the flow.

/cc @mathiasbynens

### CODE REVIEW

File: src/features/modules.md [hunk 1]
```diff
@@ -82,6 +82,8 @@ Modules are a little different from classic scripts:
 
 - The new static `import` and `export` syntax is only available within modules — it doesn’t work in classic scripts.
 
+- `await` cannot be used as a variable name anywhere in a module - variables in classic scripts can be named `await` outside of async functions.
```
Comment (mathiasbynens) [inline src/features/modules.md]:
```suggestion
- Top-level `await` is available in modules, but not in classic scripts. Relatedly, `await` cannot be used as a variable name anywhere in a module, although variables in classic scripts _can_ be named `await` outside of async functions.
```

Comment (mathiasbynens) [reply src/features/modules.md]: How about this?

Comment (jkrems) [reply src/features/modules.md]: Good to have that context in there!

**Comment by mathiasbynens** [comment]
Thanks!

**Pull request merged**